apps/web-client/src/lib/dialogs.ts

```diff
@@ -1,55 +1,66 @@
+import LL from "@librocco/shared/i18n-svelte";
+import type { TranslationFunctions } from "@librocco/shared";
+
 export interface DialogContent {
 	onConfirm: (closeDialog: () => void) => void;
 	title: string;
 	description: string;
 }
 
+let t: TranslationFunctions;
+
+LL.subscribe((LL) => {
```

Comment:
We shouldn't have a subscription without an unsubscription. I think this file was setup as an intermediary dictionary for dialog strings - do we need it anymore? Aren't we passing the same strings directly to the dialogs on the pages? If not, and we still need this, the setup needs to be rethought to avoid this loose subscription